ts/README.md

```diff
@@ -113,40 +113,43 @@ function a(data?: Array<(number|undefined)>, dOffset?: number): number {
 ```
 
 
-### Type union instead of new interface
 
-Sometimes you just need a pure merge of several classes or interfaces as a new
-type. Instead of creating an empty interface that extends them, you create a
-type union. This also works as an inline type.
+Good to Know
+------------
 
-*Incorrect:* 
-```ts
-interface A {
-    x: number;
-}
-interface B {
-    x2: number
-}
-interface C extends A, B {
-}
-```  
-*Correct:*
-```ts
-interface A {
-    x: number;
-}
-interface B {
-    x2: number
-}
-type C = (A&B);
-let example1: C = { x: 1, x2: 2 },
-    example2: (A&B) = example1;
-```
 
+### Make use of common ES6 features
 
+TypeScript does transpile most common ES6 features down to ES5-capable browsers
+like Internet Explorer 9+, Chrome 5+, Firefox 4+, Safari 5+. So you can make use
+of them:
 
-Good to Know
-------------
+* Arrow functions (without own `this` scope):
+  `(a, b) => {console.log('yeah')};`
+
+* Constants and limited variables:
+  `const yes: boolean = true; let degree: number = 180;`
```

Comment:
Is not the type assignment in `const yes: boolean = true;` redundant?

Since the type `boolean` can be inferred from the assignment, and while it is a constant then we can be certain the type does not change as well.
I guess with `let degree: number = 180;` it makes more sense to make sure the type is not changed in an later reassignment of the variable.

Reply:
Constant assignments can be dynamic as well. So the type makes sure, that yes is always `boolean` if the `true` constant is replaced by a function return for example to determine the runtime environment which results in a type list.

Reply:
Hmm I see 🤔 but if replaced by a function then the type would be inferred from the return value of the function. But I guess it has its value when you really want to enforce the type.

Reply:
An example: `const header = document.getElementById('header');` - the header const becomes type `Element|null`;